dlc/src/main/scala/org/bitcoins/dlc/sign/DLCTxSigner.scala

```diff
@@ -33,7 +32,7 @@ case class DLCTxSigner(
     builder: DLCTxBuilder,
     isInitiator: Boolean,
     fundingKey: ECPrivateKey,
-    payoutKey: ECPrivateKey,
+    finalAddress: BitcoinAddress,
```

Comment:
I'm cool with this change but would like if you added an apply that works in this way using the removed method below